extensions/interactions/ItemSelectionInput/directives/item-selection-input-interaction.directive.html

```diff
@@ -1,6 +1,6 @@
 <div class="item-selection-input-form-container">
   <form ng-submit="$ctrl.submitAnswer()">
-    <small ng-if="$ctrl.displayCheckboxes">
+    <small style="display: block; padding-bottom: 10px;">
```

Comment:
Please, don't use inline styling.

Reply:
OK, where shall I create a class for this?

Reply:
Is it ok to define a class in the respective directives? Or should I create a common class in oppia?

Reply:
Used oppia.css for now since this class is used in 2 places. PTAL!